src/management/__generated/managers/ActionsManager.ts

```diff
@@ -0,0 +1,810 @@
+/* tslint:disable */
+/* eslint-disable */
+import * as runtime from '../../runtime';
+import type {
+  GetActionVersions200Response,
+  GetActionVersions200ResponseVersionsInner,
+  GetActions200Response,
+  GetActions200ResponseActionsInner,
+  GetBindings200Response,
+  GetExecution200Response,
+  GetTriggers200Response,
+  PatchActionRequest,
+  PatchBindings200Response,
+  PatchBindingsRequest,
+  PostActionRequest,
+  PostDeployDraftVersionRequest,
+  PostTestAction200Response,
+  PostTestActionRequest,
+} from '../models';
+
+export interface DeleteActionRequest {
+  id: string;
+  force?: boolean;
```

Comment:
Can we get the doc strings for these? eg https://github.com/auth0/api2/blob/0dd85bea154866b1abf8158a2b74b0712e4ac213/api/actions/schemas/actions/delete_action.js#L15